modules/op-configuring-pipeline-resolvers.adoc

```diff
@@ -0,0 +1,49 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_content-type: CONCEPT
+[id="op-configuring-pipeline-resolvers_{context}"]
+= Configuring the pipeline resolvers
+
+:FeatureName: Resolvers
+include::snippets/technology-preview.adoc[]
```

Comment:
@ppitonak @ekristova the Resolvers feature is going to be GA with Pipelines 1.11 release. Do we need this snippet? Thanks